### ISSUE TITLE
Reproduce mini-kinetics

### ISSUE THREAD

**Issue by jnhwkim** [KaiyuYue/cgnl-network.pytorch#10]
I implemented I3D and Non-local (NL) based on the code release by Wang et al. (2018); however, CGNL got slightly lower accuracy than NL for mini-Kinetics using 8-frame and 32-frame 5-block ResNet-50 models.
Because NL is reproduced, I wonder if there is any detail for CGNL to get the scores in the paper. Is there any different hyperparameter for CGNL?

**Comment by kaiyuyue** [comment]
Hi, [warm-up](https://github.com/KaiyuYue/cgnl-network.pytorch/blob/master/train_val.py#L169-L175) could help.

**Comment by jnhwkim** [comment]
@KaiyuYue thanks for the pointer. I am afraid I've already used one and my problem seems not originated from it. I noticed you [do not scale](https://github.com/KaiyuYue/cgnl-network.pytorch/blob/master/model/resnet.py#L456) for the logit of CGNL while doing scale for NL's. Is it critical?

**Comment by kaiyuyue** [comment]
Yes. No scale factor is better for CGNL module.

**Comment by kaiyuyue** [comment]
Hi, @jnhwkim Is it working well now on kinetics ?

**Comment by jnhwkim** [comment]
@KaiyuYue thanks for the check-up. Sadly, I gave up for now. Could you share your code to examine the differences?

**Comment by kaiyuyue** [comment]
Sorry about this. I plan to reproduce the results of video classification to check it first and then release. This procedure may take some time.
